crates/update/src/cli/install.rs

```diff
@@ -0,0 +1,207 @@
+use std::io;
+
+use anyhow::Context;
+use bytes::{Buf, Bytes};
+use http_body_util::BodyExt;
+use indicatif::{ProgressBar, ProgressStyle};
+use serde::Deserialize;
+use spacetimedb_paths::SpacetimePaths;
+
+use super::ForceYes;
+
+/// Install a specific SpacetimeDB version.
+#[derive(clap::Args)]
+pub(super) struct Install {
+    /// The SpacetimeDB version to install.
+    version: semver::Version,
+
+    /// The SpacetimeDB edition(s) to install, separated by commas.
+    #[arg(long, value_delimiter = ',', action = clap::ArgAction::Set, default_value = "standalone")]
+    edition: Vec<String>,
+
+    /// Switch to this version after it is installed.
+    #[arg(long)]
+    r#use: bool,
+
+    /// The name of the release artifact to download from github.
+    #[arg(long, hide = true)]
+    artifact_name: Option<String>,
+
+    #[command(flatten)]
+    yes: ForceYes,
+}
+
+impl Install {
+    pub(super) fn exec(self, paths: &SpacetimePaths) -> anyhow::Result<()> {
+        super::tokio_block_on(async {
+            anyhow::ensure!(
+                self.edition == ["standalone"],
+                "can only install spacetimedb-standalone at the moment"
+            );
```

Comment:
wait why can we only install standalone right now? The current upgrade flow definitely allows users to install previous versions.

This is already a big PR, so I think it would be fine to leave the scope as-is but we should at least create a followup ticket.

Reply:
Standalone as opposed to cloud. I don't believe we are currently able to install cloud with `spacetime update`.

Reply:
ah gotcha that makes sense :+1: